fair_mango/metrics/metrics.py

```diff
@@ -442,17 +461,21 @@ def __call__(self) -> tuple[Sequence, list]:
                 else:
                     real_values = real_y_group
                     predicted_values = predicted_y_group
+
                 for metric_name, metric in self.metrics.items():
                     result_for_group.setdefault(metric_name, []).append(
```

Comment:
if metric_name is "sensitive" it s gonna raise an error

Reply:
Good catch! I created a check that raises a `KeyError` in case one of the metrics has the label `"sensitive"`